.jenkins/common_job_properties.groovy

```diff
@@ -86,14 +86,19 @@ class common_job_properties {
   // Sets the pull request build trigger.
   static def setPullRequestBuildTrigger(def context,
                                         def commitStatusContext,
-                                        def successComment = '--none--') {
+                                        def successComment = '--none--',
```

Comment:
things get complicated with multiple defaults, since they may force the caller to specify the default parameter just to be able to specify the following one. think about order and alternatives.